Formula/librealsense@1.12.1.rb

```diff
@@ -0,0 +1,37 @@
+class LibrealsenseAT1121 < Formula
```

Comment:
AT1

Reply:
To clarify, then the filename should be changed to librealsense@1 as well?

Reply:
yes